internal/handlers/handlers.go

```diff
@@ -51,3 +60,43 @@ func GetBigURL(shortURL string, db Storage, res http.ResponseWriter, req *http.R
 	res.Header().Set("Location", bigURL)
 	res.WriteHeader(http.StatusTemporaryRedirect)
 }
+
+func APIShortenURL(serverHost string, db Storage, res http.ResponseWriter, req *http.Request) {
+	var buf bytes.Buffer
+	_, errBody := buf.ReadFrom(req.Body)
```

Comment:
Здесь нет смысла называть ошибку по-другому, лучше просто

```suggestion
	_, err := buf.ReadFrom(req.Body)
```